.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Sat Jan 19 20:21:45 GMT 2019
```

Comment:
can we not ignore all .gradle, build or any other project related settings for IDE, by adding .gitignore file

Reply:
OK added .gradle, .idea in gitignore